### PR TITLE
Comments + Readme

### PR DESCRIPTION
**Pull request by andrasschad**
AFAIK only:
accounts.py added comments
README addition

### CODE REVIEW

**Comment by dvadlejch** [comment]
Very nice commit!

File: README.md [hunk 1]
```diff
@@ -1 +1,13 @@
-# saving_account_calculator
\ No newline at end of file
+# saving_account_calculator
+
+This will be an amazing program.
```
Comment (dvadlejch) [inline README.md:3]: I like this statement, very nice!

File: account.py [hunk 2]
```diff
@@ -3,6 +3,15 @@
 
 
 class CurrencyConverter:
+    """
+        Parameters:
+            reference_currency: The base currency, that every other currency will be converted into.
+            exchange_rates    : yes
+            
+        Functions:
+            pass():
```
Comment (dvadlejch) [inline account.py]: Still has to be filled in.

**Review by dvadlejch — CHANGES_REQUESTED** [review]
Please make the changes.

File: account.py (hunk 2 above)
Comment (andrasschad) [reply account.py]: Yes sir!

File: account.py [hunk 3]
```diff
@@ -9,7 +9,8 @@ class CurrencyConverter:
             exchange_rates    : yes
             
         Functions:
-            pass():
+            pass()
+            __init__(self): initializes the class.
```
Comment (dvadlejch) [inline account.py:13]: Cool!

**Review by dvadlejch — APPROVED** [review]
Very good, I think it can be merged to the main.

**Comment by andrasschad** [comment]
Hell Yeah, lets go!!!

**Comment by andrasschad** [comment]
Bye-bye andras branch.

**Pull request merged**